src/AWS/Orleans.Streaming.SQS/Streams/SqsStreamOptions.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.Generic;
+using Microsoft.Extensions.Options;
+
+namespace Orleans.Hosting
+{
+    public class SqsStreamOptions : PersistentStreamOptions
+    {
+        public string DeploymentId { get; set; }
```

Comment:
Should this value be configured here or should it come from some other, more central place? My understanding is that this should be replaced with `SiloOptions.ClusterId`, since we smooshed `DeploymentId` and `ClusterId` together here: https://github.com/dotnet/orleans/pull/3728

Reply:
Yes, this value should be dropped from there and inject the `SiloOptions`.  The newer providers are already doing that. Will do that on a PR.